### ISSUE TITLE
[BUG] Aria-invalid is missing for errors in inputs etc.

### ISSUE THREAD

**Issue by aniamusial** [vuestorefront/storefront-ui#2410]
**Describe the bug**
According to WCAG 3.3.1 Error Identification we should provide aria-invalid for proper error indetification.

┆Issue is synchronized with this [Jira Zadanie](https://vsf.atlassian.net/browse/SFUI-148) by [Unito](https://www.unito.io)
